### PR TITLE
Form 22-1990 Progress Bar

### PR DESCRIPTION
**Pull request by cvalarida**
The original issue (department-of-veterans-affairs/vets.gov-team#906) was to just get rid of the subway and use the segmented progress bar. Without the subway, it doesn't make sense to have the border around the content, and it's a bit off-putting to not have the title (as shown below for the 22-1995):

image

I'm not sure what to do with the headers. Sometimes they make sense (Personal Information --> Direct deposit):

image

And sometimes they're redundant (Veteran Information --> Veteran information)

image

I didn't want to over-implement without getting the green light on design, so this is what it looks like now. Suggestions?

### CODE REVIEW

File: src/js/edu-benefits/1990/containers/EduBenefitsApp.jsx
Comment (jbalboni) [inline src/js/edu-benefits/1990/containers/EduBenefitsApp.jsx]: There should be logic for this in the NavHeader component.

**Comment by jbalboni** [comment]
@cvalarida You can remove the headers on each page. We can have @goldenmeanie go through them and see if there are any that are necessary for specific pages, but the default should be that we don't show them.

File: src/js/edu-benefits/1990/containers/EduBenefitsApp.jsx
Comment (jbalboni) [inline src/js/edu-benefits/1990/containers/EduBenefitsApp.jsx]: Does this need to be in a separate row, or can it be in the same div as NavHeader?

File: src/js/edu-benefits/1990/containers/EduBenefitsApp.jsx
Comment (jbalboni) [inline src/js/edu-benefits/1990/containers/EduBenefitsApp.jsx]: Hiding this when we don't have a chapter will probably work, but the actual logic is that we don't want this to show up on the introduction and submit-message pages. You could check for that specifically, which would make what's happening clearer.

**Comment by jbalboni** [comment]
@cvalarida It probably does make sense to include a title like the 1995 form. You can use the text from the introduction page (Apply for education benefits).

**Comment by goldenmeanie** [comment]
Is origin/1990-progress-bar up to date. I'll pull it down and take a spin through to try to clear this up!

**Comment by jbalboni** [comment]
@goldenmeanie Should be.

File: src/js/edu-benefits/1990/containers/EduBenefitsApp.jsx
Comment (cvalarida) [inline src/js/edu-benefits/1990/containers/EduBenefitsApp.jsx]: It can be in the same div, but when I tried it, the NavHeader and progress bar just got squashed together:

image

I'll see what I can do to separate them while they're in the same div.

Comment (cvalarida) [inline src/js/edu-benefits/1990/containers/EduBenefitsApp.jsx]: Look good?

image

Comment (jbalboni) [inline src/js/edu-benefits/1990/containers/EduBenefitsApp.jsx]: Looks good to me!

**Comment by jbalboni** [comment]
@cvalarida It looks like there's a linting error that's causing the build check to fail above.

**Comment by cvalarida** [comment]
To sum up the changes:

---
image

---
image

---
image

**Comment by cvalarida** [comment]
Removed the border and aligned the intro page with the title to match the 22-1995.

image

---
While still keeping the indented forms.

image

**Review by jbalboni — APPROVED** [review]
Made a couple comments, but this looks good. One other thing I noticed from trying the form locally, you can remove the "Review Application" header on the review page, too.

**Comment by jbalboni** [comment]
@cvalarida Looks good to me. I think you can merge this once you fix the merge conflict (and the build passes).

**Pull request merged**
